src/utils/createEmbedding.ts

```diff
@@ -0,0 +1,54 @@
+import dotenv from 'dotenv';
+import * as path from 'path';
+import { Configuration, OpenAIApi, CreateEmbeddingRequest } from 'openai';
+dotenv.config({ path: path.resolve(__dirname, '../..', '.env.local') });
+import * as fs from 'fs';
+
+const inputChunks = JSON.parse(
+  fs.readFileSync('./src/utils//data/dummyChunks.json', 'utf8')
+);
+
+const configuration = new Configuration({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+const openai = new OpenAIApi(configuration);
+const outputArray: object[] = [];
```

Comment:
usually better to strictly define the object type rather than leave it to chance!

e.g. if you know the fields of the object define them here

Reply:
great shout